src/vs/workbench/contrib/comments/browser/commentNode.ts

```diff
@@ -604,15 +606,15 @@ export class CommentNode<T extends IRange | ICellRange> extends Disposable {
 		this._scrollableElement.setScrollDimensions({ width, scrollWidth, height, scrollHeight });
 	}
 
-	public switchToEditMode() {
+	public async switchToEditMode() {
```

Comment:
There's another caller of `switchToEditoMode` that needs to be made async. Mind if I push a commit?

Reply:
Sure! Feel free to push. Also you can take over this PR if you'd like too